package.json

```diff
@@ -59,5 +62,7 @@
       "git add"
     ]
   },
-  "dependencies": {}
+  "dependencies": {
+    "ospec": "4.0.0"
```

Comment:
why is `ospec` a dependency now? It's also already a `devDepencency` and in a different version

Reply:
seems like test runner needs it. 🤔

Reply:
Move it to dev dependencies. There *should* still be an old, runtime-deprecated version inside Mithril proper, as we have people still depending on it.